vere/sist.c

```diff
@@ -1359,6 +1359,22 @@ u3_sist_boot(void)
 
       if ( 0 != u3_Host.ops_u.key_c ) {
         u3_noun des = u3m_file(u3_Host.ops_u.key_c);
+
+        // handle trailing newline
+        //
+        {
+          c3_c* key = u3r_string(des);
+          c3_w len = strlen(key);
```

Comment:
We use hungarian-ish suffixes on variable names for all types except `u3_noun`, so this should be `key_c` and `len_w`.